config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'users#index'
+  get 'users/:id', to: 'users#show'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:id/posts/:post_id', to: 'posts#show'
```

Comment:
| @CesarHerr Routes |
|-|
| ![image](https://github.com/CesarHerr/blogApp/assets/93039359/15970695-4e79-48b4-9363-0b873bfa8d0d) |

| Microverse Routes |
|-|
| ![image](https://github.com/CesarHerr/blogApp/assets/93039359/d3f08824-1ac3-4f83-9717-2d887f31f405) |

- [ ] Great work so far but I guess you missed a route 😨 which is `/users` 🤔 You already have it as a root route but it should have a dedicated route as in  the project requirements 👍

Reply:
Hi @HAMMAS-SALEEM  Thanks for your review. I appreciated it. I did it like this because the requirements stated 'Make sure that your app handles the URLs used in the wireframes...'   the wireframe: 
![image](https://github.com/CesarHerr/blogApp/assets/111262493/dd059a19-acc8-46cd-808a-bc3cc7a02467)

But I Understand, I will change it

Reply:
Hey @CesarHerr Hope you are well. Actually, your routes file should have it as a root route and as an dedicated route as well which are mentioned saparately in the project requirements. Once in the form of wireframe and second in the routes snapshot 🤔